x-pack/plugins/secops/public/pages/hosts/index.tsx

```diff
@@ -3,66 +3,118 @@
  * or more contributor license agreements. Licensed under the Elastic License;
  * you may not use this file except in compliance with the Elastic License.
  */
+
+import { EuiBadge, EuiText } from '@elastic/eui';
 import { getOr } from 'lodash/fp';
 import React from 'react';
+import { connect } from 'react-redux';
 import { pure } from 'recompose';
+import { Dispatch } from 'redux';
+import styled from 'styled-components';
 
 import { EventItem, KpiItem } from '../../../common/graphql/types';
-import { BasicTable, Columns } from '../../components/basic_table';
+import { BasicTable } from '../../components/basic_table';
 import { HorizontalBarChart, HorizontalBarChartData } from '../../components/horizontal_bar_chart';
 import { Pane1FlexContent } from '../../components/page';
 import { Placeholders, VisualizationPlaceholder } from '../../components/visualization_placeholder';
 import { EventsQuery } from '../../containers/events';
+import { timelineActions } from '../../store';
+
+const startDate = 1521830963132;
+const endDate = 1521862432253;
+// const startDate = 1541044800000;
+// const endDate = 1543640399999;
```

Comment:
Optional since we're right at the end of this fast code run...

Two lines of commented out code here. Remove it or document why they're here (temp testing).

Reply:
that's for the demo tomorrow, to show good data for the timeline and the other one for kPI
let me add a comment